version.txt

```diff
@@ -1 +1 @@
-1.0
\ No newline at end of file
+2.0
```

Comment:
Do we need a major version change? I would think that making a required field optional would only need a minor version change. I don't really care too much though.

Reply:
Do we ?  Probably not.  I am planning on a few more changes and just thought to bump it to 2